python/ray/tests/test_metrics_agent.py

```diff
@@ -60,6 +55,24 @@
     "ray_registered_placement_group",
     "ray_infeasible_placement_group",
     "ray_new_resource_creation_latency_ms_sum",
+    "ray_grpc_server_req_process_time_ms",
+    "ray_grpc_server_req_new_total",
+    "ray_grpc_server_req_handling_total",
+    "ray_grpc_server_req_finished_total",
```

Comment:
Based on my debugging experience, not separating them produces better experience since we already have so many labels (per handler). Mixing them with 3 additional labels probably make initial graph really hard to interpret, and we will always create 3 separate graphs anyway to interpret it.

Reply:
But I have no strong preference.

Reply:
No strong opinion, since we should remove this once we have operation stats by default, as this will be duplicated.